source/declarations.tex

```diff
@@ -1954,6 +1954,20 @@
 that contains a placeholder type,
 \tcode{T} is the declared type of the non-type template parameter
 and $E$ is the corresponding template argument.
+\item
+For an explicit type conversion\iref{expr.type.conv},
+\tcode{T} is the specified type, which shall be \keyword{auto}.
+\begin{itemize}
+\item
+If the initializer is a \grammarterm{braced-init-list},
+it shall consist of a single brace-enclosed \grammarterm{assignment-expression}
+and $E$ is the \grammarterm{assignment-expression}.
+\item
+If the initializer is a parenthesized \grammarterm{expression-list},
+the \grammarterm{expression-list} shall be
+a single \grammarterm{assignment-expression}
+and $E$ is the \grammarterm{assignment-expression}.
```

Comment:
It's weird to say this explicitly for `auto(4)` but not for `auto x(4);`, since the latter's initializer is `(4)`.  It's preexisting, but should we adjust /2.2 (or even try to combine them)?

Reply:
But... we say that in p2.2.3:
> If the initializer is a parenthesized expression-list, the expression-list shall be a single assignment-expression and E is the assignment-expression

Reply:
I'd say let's leave this for now? I'm not sure what exactly we could improve here.